packages/eslint-plugin/package.json

```diff
@@ -42,6 +42,7 @@
   },
   "dependencies": {
     "@typescript-eslint/experimental-utils": "2.23.0",
+    "eslint": "^6.8.0",
```

Comment:
I need `eslint` because I import a list from it
https://github.com/typescript-eslint/typescript-eslint/pull/1739/files#diff-98976b0a1b01edcf278c08cbf1cb4bc8R1
It seems a bit overkill, so I am considering copypasting this list inside my own file. What do you think?